manticore/core/cpu/abstractcpu.py

```diff
@@ -363,17 +369,15 @@ class Cpu(Eventful):
 
     def __init__(self, regfile, memory, **kwargs):
         assert isinstance(regfile, RegisterFile)
+        disasm = kwargs.pop("disasm", None)
         super(Cpu, self).__init__(**kwargs)
         self._regfile = regfile
         self._memory = memory
         self._instruction_cache = {}
         self._icount = 0
         self._last_pc = None
-
-        self._md = Cs(self.arch, self.mode)
-        self._md.detail = True
-        self._md.syntax = 0
-
+        if not hasattr(self, "disasm") or self.disasm is None:
+            self.disasm = init_disassembler('capstone', self.arch, self.mode)
```

Comment:
I don't think it should be a property, I just think it could be a private variable, since it is never accessed outside the Cpu class.